README.md

```diff
@@ -1,6 +1,6 @@
 # PASS - Personal Access System for Services
 
```

Comment:
Could we have the images be placed within this repo instead?

Reply:
Oops, wrong line. Was meant to be Line 3 with the assets.

Reply:
does sourcing from the PASS wiki work for you? If not I can add the light/dark mode to the assets folder of PASS itself and source from there assuming the in app logo will get updated. Either way the same image is used in the wiki and PASS.

Reply:
Yeah, I think that should be fine.